dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/configuration/BackPressureMitigation.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.core.configuration
+
+/**
+ * Defines the mitigation to use when a queue hits the maximum back pressure capacity.
+ */
+enum class BackPressureMitigation {
+
+    /**  Drop the oldest items already in the queue to make room for new ones. */
+    DROP_OLDEST,
```

Comment:
It'll be the latter as each addition is independent. We don't have a way to "add n items" to the queue, it will always be 5⨉"add one item", which mean each one will be resolved independently.